mocha-client-tests/tests/lib/background.test.js

```diff
@@ -0,0 +1,9 @@
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function() {
+            Background.ping(false, false, function(response) {
+                response.should.equal('pong');
```

Comment:
I don't see the change to make this `expect()`. Do you have any unpushed commits maybe?